src/mockdata.js

```diff
@@ -0,0 +1,55 @@
+export const mockSightings = [
+  {
+    name: "",
+    lat: "47.7511",
+    lng: "-120.7401",
+    description: "BIG BLUE BALL IN SKY OMG",
+    eventType: "sighting",
+    image: "https://cdn.mos.cms.futurecdn.net/QkuLCPzH7GUVS9MthdWM9M.jpg",
```

Comment:
I'm curious as to how we're going to allow users to upload their own images, rather than a link to a url.  I'm assuming there's a package for that?